### ISSUE TITLE
Seeding

### ISSUE THREAD

**Issue by Que-tin** [CSS-Next/logo.css#8]
- wikimedia
- icones.js.org
- Smashing / CSS-Tricks?
- Brands of the World
- PR to vsicons
- Bikeshed / CSS-WG

**Comment by romainmenke** [comment]
GitHub topic for CSS: https://github.com/github/explore/tree/main/topics/css

**Comment by argyleink** [comment]
- vscode icon packs

**Comment by argyleink** [comment]
> Should this happen before [w3c/csswg-drafts#11193](https://github.com/w3c/csswg-drafts/issues/11193) got resolved?

i dont think it needs to wait. maybe it'd give it more klout for the folks we introduce it to, but they likely adopted the community JS logo without Javascript ™️ 's approval. but i dont think this logo is klout driven, it's community driven. wherever folks wants to use it, we're happy this is serving that purpose for them ❤️ 

what were you thinking? waiting for it to be something more "official"?

**Comment by Que-tin** [comment]
Not exactly to be more "official" rather than to make sure there aren't small things the CSSWG points out that might need to be adjusted after seeding.

Would be annoying to reiterate through all of places to update the logo again, no?

**Comment by argyleink** [comment]
i see, you're thinking there might be change requests still, gotcha!
